app/app.cpp

```diff
@@ -1,11 +1,54 @@
 #include "app.h"
-#include <format>
 
 namespace vsite::oop::v1
 {
-/*
-* function implementations
-*/
+	std::string to_hex(int broj)
+	{
 
-} // namespace
+		return std::format("{:X}", broj);
+	}
+
+	std::string to_exp(double broj)
+	{
+				
+		return std::format("{:.2e}", broj);
+	}
+
+	void mult_table(int broj, std::ostream& os)
+	{
+		if (broj == 0 || broj > 20)
+		{
+			os << "";
```

Comment:
Ovo ne treba, ovo ne radi ništa.

Reply:
Ne prolazi mi prvi i zadnja 2 testa bez toga

Reply:
Slažem se da morate provjeriti je li broj van zadanih vrijednosti. Ali, ne morate u tom slučaju slati na ispis prazan string (što bi to trebalo napraviti?) - `return;` je dovoljno.